### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Import Bookmarks

### DISCUSSION THREAD

**Discussion by ach5910** [dailydotdev/daily#1010] [Feature Requests]
### Description

I would like to import my books marks from Medium into daily dev.

### Solution

In my profile I should be allowed to link accounts from other blog/article platforms, like Medium, via Oauth and allow permission for daily dev to read my books marks and import them.

### Benefit

Daily dev is already a hub for curated content across different platforms but account linking would allow daily dev to be my one-stop-shop.

Additionally if opening an article results in switching to that platforms application (which I prefer over WebViews) I shouldn't have to bookmark the article twice in both apps/platforms.

### Feedback

Love the app, and very impressed with the PWA. Also the idea of a reputation point system centered around articles read over articles written peaks my interest.

**Comment by idoshamun** [comment]
Hey @ach5910,

This is a great suggestion.
Unfortunately, at the moment we have technical issues with bookmarking articles that are not part of our feed. And obviously, we cannot let every article that someone bookmarked get into the feed. We have strict criteria on what's going into the feed to keep the quality level.
Of course, it doesn't mean that it is a "no go" but just means that it might take some time.
I will keep this issue posted so we can refer back to it once in a while in our product meetings.
Please note that we are a small team and I'm the only developer 🙏

As for your feedback, I love it! daily.dev is all about delighting the readers where all the other platforms focus on the writers.
Thank you for dedicating the time, it's highly appreciated.

**Comment by EO4wellness** [comment]
Daily.Dev is great!  I would also add, if anyone has the time to work on this in the future, a way to add articles and bookmarks from LinkedIn.  I'm always seeing great content there, which is not here, but LinkedIn isn't intuitive on finding those items again and their "feed' is too large to successful search for the item(s) when needed.

**Comment by sshanzel** [comment]
Hey @EO4wellness, glad to know you are enjoying the app. For the bookmarks, it is not yet possible, but we allow submitting content that is not yet in the app. You can check the docs page below to know more about it. And the great part is that you will earn reputation points for the engagements in that article!

https://docs.daily.dev/docs/key-features/community-picks

**Comment by worldwidejamie** [comment]
To add to the conversation, I would appreciate the ability to add bookmarks generally and not necessarily from any specific service. The curated/vetted content is important, so not wanting to flood the feed with everyone's saved sites is understandable.

I use daily.dev as it's name suggests I should; daily. It's also one of the few apps I use across all of my devices. Instead of trying to integrate a new tool into my workflow, taking advantage of the tool I already depend of for a similar functionality would be excellent.
